src/configuration.c

```diff
@@ -0,0 +1,55 @@
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+#include "../include/configuration.h"
+/*
+    Gibt SUCCESS oder eine Fehlermeldung über 1000 zurück
+*/
+enum Position {key, value};
+
+const char PORT_KEY[] = "PORT";
+const char PATH_KEY[] = "PATH";
+const char MAX_SESSIONS_KEY[] = "MAX_SESSIONS";
+
+Configuration config = {5678, "./data", 5};
+
+int loadConfig(char path[]) {
+    char c;
+    FILE *fp;
+    fp = fopen(path, "r");
+    if (fp == NULL) {
+        printf("An error occurred while opening the config file. The application will stick to its default config.\n");
+        return 1001;
+    }
+    char keyContent[128] = "";
+    char valueContent[128] = "";
```

Comment:
The length of valueContent does not match the maximum length of a string in the Configuration struct(512 at the moment).